### PR TITLE
Add FileView, Study, and terms for light sheet microscopy dataset

### PR DESCRIPTION
**Pull request by Bankso**
## Changelog

 - Added FileView and Study attributes and schema definitions
 - Added Zebrafish as species
 - Added DAE as dataset format
 - Added Multiscale Light Sheet Microscopy as an assay
 - Add Component Keys with unique validation rule

### CODE REVIEW

**Comment by Bankso** [comment]
@aditigopalan could you take a quick look at this PR when you have a second? I've already generated a manifest based on these additions, so I think it should be good to go. I'd like to get this FileView model merged so I can add it to our data model docs

**Review by aditigopalan — APPROVED** [review]
Lgtm!

File: docs/valid_values/file.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+List of standard terms for the [Dataset Sharing Plan Model](../model/file.md).
```
Comment (vpchung) [inline docs/valid_values/file.md:1]:
```suggestion
List of standard terms for the [File Model](../model/file.md).
```

Comment (vpchung) [reply docs/valid_values/file.md:1]: Whoops, made a suggestion without realizing the PR has already been merged.

I will file this as a bug instead!

**Pull request merged**